### DISCUSSION TITLE
Geotiff-zoom level

### DISCUSSION THREAD

**Discussion by paolodep36** [opengeos/segment-geospatial#97] [Q&A] [answered]
I wanted to ask if there was an option to specify a zoom level when I pass one of my geotiff. I noticed in the meta ai site demo that if I try to segment the image with different zoom levels, Sam is able to segment some objects better.

**Comment by giswqs** [comment, marked as answer]
Currently, the answer is no. The program convert the image to numpy array for segmentation. If you want different image resolution, you can other third party libraries (e.g., scikit-image) to manipulate the image.
